unit_test/test_spdm_responder/measurements.c

```diff
@@ -2507,6 +2513,72 @@ void libspdm_test_responder_measurements_case34(void** state)
 #endif
 }
 
+/**
+ * Test 35: Successful response V1.3 to get a number of measurements without signature
+ * Expected Behavior: get a RETURN_SUCCESS return code, correct context field
+ **/
+void libspdm_test_responder_measurements_case35(void **state)
+{
+    libspdm_return_t status;
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    size_t response_size;
+    uint8_t response[LIBSPDM_MAX_SPDM_MSG_SIZE];
+    spdm_measurements_response_t *spdm_response;
+    uint8_t *requester_context;
+    uint8_t *response_contest;
+
+    spdm_test_context = *state;
+    spdm_context = spdm_test_context->spdm_context;
+    spdm_test_context->case_id = 35;
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_13 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_AUTHENTICATED;
+    spdm_context->local_context.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEAS_CAP_SIG;
+    spdm_context->connection_info.algorithm.base_hash_algo =
+        m_libspdm_use_hash_algo;
+    spdm_context->connection_info.algorithm.base_asym_algo =
+        m_libspdm_use_asym_algo;
+    spdm_context->connection_info.algorithm.measurement_spec =
+        m_libspdm_use_measurement_spec;
+    spdm_context->connection_info.algorithm.measurement_hash_algo =
+        m_libspdm_use_measurement_hash_algo;
+    libspdm_reset_message_m(spdm_context, NULL);
+
+    libspdm_secret_lib_meas_opaque_data_size = 0;
+
+    response_size = sizeof(response);
+
+    requester_context = ((uint8_t *)&m_libspdm_get_measurements_request17) +
+                        m_libspdm_get_measurements_request17_size;
+    libspdm_get_random_number(SPDM_NONCE_SIZE, requester_context);
+    m_libspdm_get_measurements_request17_size += SPDM_REQ_CONTEXT_SIZE;
```

Comment:
`libspdm_get_random_number(SPDM_NONCE_SIZE, requester_context);`
1. why fill a random_number into `requester_context` here? 
2. `SPDM_NONCE_SIZE` is 32, how to fill into `requester_context`?  due to `SPDM_REQ_CONTEXT_SIZE` is just 8.